tests/components/deconz/test_climate.py

```diff
@@ -72,6 +73,7 @@ async def test_climate_devices(hass):
     assert "climate.presence_sensor" not in gateway.deconz_ids
     assert "climate.clip_thermostat" not in gateway.deconz_ids
     assert len(hass.states.async_all()) == 3
+    assert len(gateway.entities[climate.DOMAIN]) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)